src/shafaque/Assignment78_1.java

```diff
@@ -0,0 +1,20 @@
+package shafaque;
+
+public class Assignment78_1 {
+
+	void getNumerPtternPrint() {
+		for (int i = 1; i <= 6; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible